src/modules/endpoint/endpoints/categories.js

```diff
@@ -12,7 +12,9 @@ function getUrlWithoutCity (url) {
 }
 
 export default new EndpointBuilder('categories')
-  .withStateToUrlMapper(state => `https://cms.integreat-app.de/${getCurrentCityConfig().cmsName}/de/wp-json/extensions/v0/modified_content/pages?since=1970-01-01T00:00:00Z`)
+  .withStateToUrlMapper(state => {
+    return `https://cms.integreat-app.de/${getCurrentCityConfig(state.cityConfigs._data).cmsName}/de/wp-json/extensions/v0/modified_content/pages?since=1970-01-01T00:00:00Z`
```

Comment:
Is there any other way to retrieve the cityConfigs from the state than `state.cityConfigs._data`?

Reply:
As far as I can see, with the current setup of the withFetcher: no.